rpyc/lib/win_unix_socket.py

```diff
@@ -0,0 +1,535 @@
+"""
+Impersonate a Unix domain socket using ... a Unix domain socket.
+
+Microsoft added support for AF_UNIX to Windows 10 in early 2018.
+There's no python support as of Python 3.8: bpo-33408.
+
+This file is a partial reimplementation of CPython's socketmodule.c
+
+What's here is just what's needed for RPyC itself, and there's probably less
+error-handling.
+
+See also rpyc.lib.compat which is responsible for defining unix_socket to point
+either to socket.socket or to win_unix_socket.WindowsUnixSocket depending on
+the platform.
+"""
+from __future__ import division
+import ctypes
+import ctypes.wintypes
+import socket
+import time
+import sys
+
+socket.AF_UNIX = 1
+POLLIN = 0x100 | 0x200
+POLLOUT = 0x10
+WSAEINTR = 10004
+FIONBIO = 0x8004667e
+MAX_PATH_LENGTH = 260
+WSAEWOULDBLOCK = 10035
+SOL_SOCKET = 65535
+SO_ERROR = 4103
+EISCONN = 10056
+WSA_INVALID_PARAMETER = 87
+FORMAT_MESSAGE_ALLOCATE_BUFFER = 0x100
+FORMAT_MESSAGE_FROM_SYSTEM = 0x1000
+FORMAT_MESSAGE_IGNORE_INSERTS = 0x200
+FORMAT_MESSAGE_FLAGS = (FORMAT_MESSAGE_ALLOCATE_BUFFER
```

Comment:
Can you PEP-8 lint this? I think this line is wrong but I'm not sure.

Reply:
I will